### ISSUE TITLE
feat(py): Add BatchNormalization ONNX node translator

### ISSUE THREAD

**Issue by davidrsch** [davidrsch/orbital-py-fork#17]
## Summary

`BatchNormalization` is one of the most widely used layers in modern neural networks (ResNets, EfficientNet, modern tabular MLPs). It is a first-class ONNX node type and is emitted by both PyTorch and keras3 exporters.

At inference time BatchNorm has no batch statistics — it uses running mean and variance (stored as initializers). The formula is exact and fully expressible in SQL:

```
y = ((x - running_mean) / sqrt(running_var + epsilon)) * gamma + beta
```

Currently `orbital.translate()` raises `KeyError` for `BatchNormalization` nodes.

## Acceptance Criteria

- `BatchNormalizationTranslator` implemented in `src/orbital/translation/steps/batchnorm.py`
- Reads five initializers: `scale` (γ), `bias` (β), `input_mean` (μ), `input_var` (σ²), `epsilon` attribute (default `1e-5`)
- Formula applied per-feature: `((x_i - mean_i) / sqrt(var_i + eps)) * scale_i + bias_i`
- Returns a `ValueVariablesGroup` with the same number of columns as input
- Registered as `"BatchNormalization"` in `TRANSLATORS`
- Unit tests: registered, basic 2-feature correctness, `epsilon` edge case

## Notes

- ONNX spec: inputs are `[X, scale, bias, input_mean, input_var]`; outputs `[Y]` (training outputs `mean`, `var` omitted at inference)
- Opset 9+: `training_mode` attribute defaults to `0` (inference)
- Reference: https://onnx.ai/onnx/operators/onnx\_\_BatchNormalization.html
- This pairs well with the existing `Relu` and `Gemm` translators to support the full ResNet block: `Gemm → BatchNorm → Relu`

**Comment by davidrsch** [comment]
Implemented in commit `199848b` on `feat/ann-mlp-support`.\n\n- `BatchNormalizationTranslator` added in `src/orbital/translation/steps/batchnorm.py`\n- Reads 5 inputs: `X`, `scale` (γ), `bias` (β), `input_mean` (μ), `input_var` (σ²)\n- `epsilon` attribute defaulting to `1e-5`\n- Formula: `((x_i - mean_i) / sqrt(var_i + eps)) * scale_i + bias_i`\n- Registered as `\"BatchNormalization\"` in `TRANSLATORS`\n- Unit tests: registered, 2-feature correctness, epsilon edge case
